packages/coil-extension/src/background/services/Stream.ts

```diff
@@ -180,13 +180,8 @@ export class Stream extends EventEmitter {
     // is severed before full establishment
     if (!this._active) throw new Error('aborted monetization')
 
-    const tier = await this._tiers.getTier(this._initiatingUrl)
-    const serverWithTier = `${this._server}?tier=${tier}`
-
-    if (!this._active) throw new Error('aborted monetization')
-
     this._plugin = new IlpPluginBtp({
-      server: serverWithTier,
+      server: this._server,
```

Comment:
Do you want me to kill the BandwidthTiers class entirely. It's no longer going to be used?